src/calendar/day/marking/style.ts

```diff
@@ -26,6 +26,7 @@ export default function styleConstructor(theme: Theme = {}) {
       borderBottomRightRadius: 2,
       marginRight: 4
     },
-    ...(theme.stylesheet?.marking || {})
+    //@ts-ignore
+    ...(theme['stylesheet.marking '] || {})
```

Comment:
Probably shouldn't be a space at the end of 'stylesheet.marking '?

Reply:
True, thanks for pointing that out.